error/src/main/java/eu/europeana/api/commons_sb3/error/exceptions/InternalServerException.java

```diff
@@ -0,0 +1,24 @@
+package eu.europeana.api.commons_sb3.error.exceptions;
```

Comment:
Not sure if this is needed as we already throw 500 Internal server error if we can not find the status in EuropeanaApiException. This package is only for common exceptions to be used by other apis